packages/core/src/App/Containers/Layout/header/trading-hub-header.jsx

```diff
@@ -45,8 +45,26 @@ export const TradersHubHomeButton = ({ is_dark_mode }) => {
     );
 };
 
-const TradingHubOnboarding = ({ is_dark_mode, setIsOnboardingVisited }) => {
-    const history = useHistory();
+const TradingHubOnboarding = ({
+    is_dark_mode,
+    toggleIsWalletTourOpen,
+    switchAccount,
+    is_wallet_switching,
+    // setIsOnboardingVisited,
+}) => {
+    // const history = useHistory();
+
+    const { data } = useWalletsList();
+    const first_loginid = data?.[0].loginid;
```

Comment:
@aizad-deriv we don't rely on sorted loginIds, find the USD wallet and see if it's selected one.